File: Java/Program/PetShop.java
// membuat class PetShop
public class PetShop{
    // variabel yang digunakan
    private int id;
    private String nama;
    private String kategori;
    private int harga;

    // constructor
    public PetShop(){
    }

    public PetShop(int id, String nama, String kategori, int harga){
        this.id = id;
        this.nama = nama;
        this.kategori = kategori;
        this.harga = harga;
    }

    // setter dan getter

    // setter id
    public void setId(int id){
        this.id = id;
    }

    // setter nama produk
    public void setNama(String nama){
        this.nama = nama;
    }

    // setter kategori
    public void setKategori(String kategori){
        this.kategori = kategori;
    }

    // setter harga
    public void setHarga(int harga){
        this.harga = harga;
    }

    // getter id
    public int getId(){
        return id;
    }

    // getter nama produk
    public String getNama(){
        return nama;
    }

    // getter kategori
    public String getKategori(){
        return kategori;
    }

    // getter harga
    public int getHarga(){
        return harga;
    }

    // method untuk menampilkan data
    public void tampilkanData(){
        System.out.println("ID         : " + id);
        System.out.println("Nama       : " + nama);
        System.out.println("Kategori   : " + kategori);
        System.out.println("Harga      : " + harga);
    }
}

File: Python/Program/PetShop.py
# membuat class PetShop
class PetShop:
    # constructor
    def __init__(self, id=None, nama=None, kategori=None, harga=None):
        # variabel yang digunakan
        self.id = id
        self.nama = nama
        self.kategori = kategori
        self.harga = harga

    # setter dan getter

    # setter id
    def set_id(self, id):
        self.id = id

    # setter nama produk
    def set_nama(self, nama):
        self.nama = nama

    # setter kategori
    def set_kategori(self, kategori):
        self.kategori = kategori

    # setter harga
    def set_harga(self, harga):
        self.harga = harga

    # getter id
    def get_id(self):
        return self.id

    # getter nama produk
    def get_nama(self):
        return self.nama

    # getter kategori
    def get_kategori(self):
        return self.kategori

    # getter harga
    def get_harga(self):
        return self.harga

    # method untuk menampilkan data
    def tampilkan_data(self):
        print(f"ID         : {self.id}")
        print(f"Nama       : {self.nama}")
        print(f"Kategori   : {self.kategori}")
        print(f"Harga      : {self.harga}")